builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -49,6 +49,25 @@ func TestAccAWSDBInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstance_withoutEngineVersion(t *testing.T) {
```

Comment:
meaning let's not include this test, instead just remove `engine_version` from `testAccAWSDBInstanceConfig` that's used in `TestAccAWSDBInstance_basic` and we'll be good